src/main/java/com/mycompany/myapp/web/rest/reservation/ReservationResource.java

```diff
@@ -0,0 +1,55 @@
+package com.mycompany.myapp.web.rest.reservation;
+
+import com.codahale.metrics.annotation.Timed;
+import com.mycompany.myapp.domain.Reservation;
+import com.mycompany.myapp.service.reservation.ReservationService;
+import com.mycompany.myapp.web.rest.util.HeaderUtil;
+import io.github.jhipster.web.util.ResponseUtil;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/reservation")
+public class ReservationResource {
+
+    ReservationService reservationService;
+
+    public ReservationResource(ReservationService reservationService){
+        this.reservationService = reservationService;
+    }
+
+    /**
+     * GET  /reserve : make reservation for pool
+     *
+     */
+    @GetMapping("/reserve")
+    @Timed
+    public ResponseEntity<Void> reserve(@RequestParam(value = "poolId") String poolId, @RequestParam(value = "count") int count) throws Exception {
+        reservationService.reserve(poolId, count);
+        return ResponseEntity.ok().headers(HeaderUtil.createAlert( "reservation.deleted", poolId)).build();
+    }
+
+    /**
+     * GET  /get : make reservation for pool
```

Comment:
M8 please delete those "java docs", they are confusing af

Reply:
sure copy paste is bad but fast